tuf/ngclient/updater.py

```diff
@@ -185,8 +185,12 @@ def download_target(
         else:
             target_base_url = _ensure_trailing_slash(target_base_url)
 
-        target_filepath = targetinfo["filepath"]
         target_fileinfo: "TargetFile" = targetinfo["fileinfo"]
+        target_filepath = targetinfo["filepath"]
+        if self.config.consistent_snapshot:
```

Comment:
Good suggestion.
There are cases like PyPi and Warehouse when one could use `consistent_snapshot` without using `prefix_filename_with_hash`.
I will only use the name `prefix_targets_with_hash` because it's clearer that it's meant for targets.